h/static/scripts/test/frame-sync-test.js

```diff
@@ -52,8 +53,14 @@ describe('FrameSync', function () {
   });
 
   beforeEach(function () {
-    fakeAnnotationUI = fakeStore({annotations: []});
-    fakeAnnotationUI.updateAnchorStatus = sinon.stub();
+    fakeAnnotationUI = fakeStore({annotations: []}, {
```

Comment:
The core `(state, action) => state` function has no side effects, but the actual dispatch function does because of [middleware](http://redux.js.org/docs/advanced/Middleware.html) which does things like trigger an Angular `$digest` cycle after each update (and in future, we may also use this as a place to capture metrics, capture errors etc.)